diagram.md

```diff
@@ -191,28 +193,39 @@ The `Cluster Elements` contain all of the video frames and audio for each track.
 ```
 Figure: Representation of a `Cluster Element` and its immediate `Child Elements`.
 
-Below is a representation of the Block structure.
+```
++---------------------------------------------+
+| Block | Portion of | Data Type   | Bit Flag |
+|       | a Block    |             |          |
+|       |-------------------------------------+
+|       | Header     | TrackNumber |
+|       |            |-------------|
+|       |            | Timecode    |
+|       |            |------------------------+
```

Comment:
Not too sure about the extra column. It's confusing.
You could just put Flags and below: Gap (1 bit), lacing (1 bit).